core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -235,7 +235,8 @@ class SocketServerTest extends JUnitSuite {
     val trustStoreFile = File.createTempFile("truststore", ".jks")
     val overrideProps = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, enableSsl = true,
       trustStoreFile = Some(trustStoreFile))
-    overrideProps.put("listeners", "SSL://localhost:0")
+    overrideProps.put(KafkaConfig.ListenersProp, "SSL://localhost:0")
+    overrideProps.put(KafkaConfig.InterBrokerSecurityProtocolProp, SecurityProtocol.SSL)
```

Comment:
Yes, this was misconfigured based on the validations in this patch. listeners=SSL, but by default security.inter.broker.protocol=PLAINTEXT which is not valid since there are not PLAINTEXT listeners.